ibm_mq/datadog_checks/ibm_mq/config.py

```diff
@@ -89,12 +103,16 @@ def __init__(self, instance):
         )  # type: Dict[str, str]
 
         custom_tags = instance.get('tags', [])  # type: List[str]
-        self.tags_no_channel = [
+        tags = [
             "queue_manager:{}".format(self.queue_manager_name),
-            "mq_host:{}".format(self.host),  # 'host' is reserved and 'mq_host' is used instead
-            "port:{}".format(self.port),
-        ] + custom_tags  # type: List[str]
-        self.tags = self.tags_no_channel + ["channel:{}".format(self.channel)]  # type: List[str]
+            "connection_name:{}".format(self.connection_name),
+        ]  # type: List[str]
+        tags.extend(custom_tags)
+        if host or port:
```

Comment:
?

```suggestion
        if host and port:
```

Reply:
Actually, either host/port are both **not defined**, or both are **defined**. Only one being set shouldn't happen.

That said, `host or port` is a bit safer, it's better to have one of the tags with `None` than no tag at all.

Reply:
Yep, okay!